setup.cfg

```diff
@@ -31,37 +31,37 @@ setup_requires =
       pytest-runner>=2.0,<3dev
 # Add here dependencies of your project (semicolon/line-separated), e.g.
 install_requires =
-      pymongo~=3.12.2
-      motor>=2.0
       aiocache
-      aioipfs>=0.4.7
-      aiohttp>=3.3.2,<4.0.0a1
-      aiohttp-session[secure]
+      aiocache~=0.11.1
       aiohttp-jinja2
+      aiohttp-session[secure]
+      aiohttp>=3.3.2,<4.0.0a1
       aiohttp_cors
-      configparser
+      aioipfs>=0.4.7
+      aleph-client>=0.4.1
+      coincurve<16.0.0
       configmanager
-      uvloop
-      web3
-      eth_account
+      configparser
+      cosmospy
+      dataclasses_json>=0.5.2
+      eth_account<0.6.0
+      eth-rlp==0.2.1
```

Comment:
I had to peg this one to a known version, there seemed to be a few issues with the pip dependency resolver otherwise. I'll try to remove it now that the code is cleaned up, otherwise we can certainly remove it safely in #138 .